pkg/controller/broadcastjob/broadcastjob_controller.go

```diff
@@ -178,7 +175,8 @@ func (r *ReconcileBroadcastJob) Reconcile(_ context.Context, request reconcile.R
 	addLabelToPodTemplate(job)
 
 	if IsJobFinished(job) {
-		if isPast, leftTime := pastTTLDeadline(job); isPast {
+		isPast, leftTime := pastTTLDeadline(job)
```

Comment:
Why you need to change here?

Reply:
I wanted to write something like 

```go
if !isPast && leftTime>0 { return xxx }

klog.Infof(xxx)
...
return xxx
```

But I was not sure of its correctness, so I reverted it, but this modification was kept.